lib/mergePack.js

```diff
@@ -0,0 +1,25 @@
+import merge from 'package-merge';
```

Comment:
You can simply use https://lodash.com/docs#merge which is recursive.

https://www.npmjs.com/package/lodash.merge would work fine.